docs/pages/access-controls/reference.mdx

```diff
@@ -206,7 +206,7 @@ The following variables can be used with `logins` and `windows_desktop_logins` f
 
 | Variable | Description |
 | - | - |
-| `{{internal.logins}}` | Substituted with "allowed logins" parameter used in `tctl users add [user] <allowed logins>` command. This applies only to users stored in Teleport's own local database. |
+| `{{internal.logins}}` | This applies users stored in Teleport's own local database and users from a root cluster.  For local users that will substitute in the "allowed logins" parameter used in `tctl users add [user] <allowed logins>` command.   If the role is within a leaf cluster in [Trusted Clusters](../setup/admin/trustedclusters.mdx) the logins from the root cluster will also apply. |
```

Comment:
```markdown
This applies users stored in Teleport's own local database and users from a root 
cluster.
```

What does "applies users" mean here? I would be more specific about what's happening at the level of the template variable.

```markdown
If the role is within a leaf cluster in [Trusted Clusters]
(../setup/admin/trustedclusters.mdx) the logins from the root cluster will also 
apply.
```
Does "apply" mean "be substituted for the variable"? Also, what does "logins" mean here? The login a user assumes when running `tsh ssh`?

Reply:
@ptgott please take a look at the updated language